### ISSUE TITLE
Multiple DB instance on single Docker block CPU core

### ISSUE THREAD

**Issue by praveshtora** [docker-library/redis#235]
I am having two microservices using the same redis server running via docker. Both of then are configured to use different db i.e. db0 and db1 in redis. However after some time I am observing both of the CPU core are being used 100% by redis.

**Comment by wglambert** [comment]
Can you give all the commands you ran and any relevant files or logs for reproducing the issue

**Comment by praveshtora** [comment]
I am using [redis](https://www.npmjs.com/package/redis) npm to make connection to redis server and in its configuration is something like this - 

const RedisClient = Redis.createClient({host: <dockername>, db: 0}) from microservice 1 and
const RedisClient = Redis.createClient({host: <dockername>, db: 1}) from microservice 2

I am not sure if this is the cause of issue, but this was the only change that I brought in already running server. I am observing this issue on two my amazon EC2 server instance after making this change.

The docker image for redis in 4.0.9.

![Screenshot from 2020-04-16 09-28-11](https://user-images.githubusercontent.com/20622998/79413560-e118b880-7fc5-11ea-9aab-5ad77da90b03.png)

**Comment by tianon** [comment]
See https://github.com/docker-library/docs/pull/1571/files

(duplicate of https://github.com/docker-library/redis/issues/217, #225, and probably others)

TL;DR, you exposed your Redis server(s) publicly to the internet without a password, and they are now compromised.  I'd suggest throwing those containers away completely and creating fresh containers with a password configured and/or without exposing the ports to the internet.
